src/main/waveform/waveform.core.js

```diff
@@ -273,6 +273,10 @@
       }, 0);
     };
 
+    if (isXhr2 && this.peaks.withCredentials) {
```

Comment:
The `withCredentials` flag should be part of the `options` object passed to `Peaks.init()`, rather than being set on the `peaks` object itself. Also, a default value (`false`) should be added to the `options` object in [main.js](https://github.com/bbc/peaks.js/blob/master/src/main.js#L56).

Reply:
Would you like me to add a @since tag to the `withCredentials` docblock in main.js?